client/src/components/Landing.js

```diff
@@ -24,15 +24,15 @@ function Landing() {
         </div>
         <div className="column small-12 large-4">
           <h1>1</h1>
-          <p className="text-left">Activity Director Schedules Open Spots for Musicians</p>
+          <img src="../../public/img/how/how_organizer.png" alt="Organizers Create Volunteering Spots" />
```

Comment:
Huh, interesting. I'm surprised this works. I thought we'd have to just use `/public/...` for the route, because that is how it serves the static files.